src/bin/util/softhsm2-util.cpp

```diff
@@ -1205,6 +1248,98 @@ int importSecretKey(char* filePath, CK_SLOT_ID slotID, char* userPIN, char* labe
 	return result;
 }
 
+// Import a certificate from given path
+int importCertificate
+(
+	char* filePath,
+	CK_SLOT_ID slotID,
+	char* userPIN,
+	char* label,
+	char* objectID,
+	int forceExec
+)
+{
+	char user_pin_copy[MAX_PIN_LEN+1];
```

Comment:
Initializing this array to zero `= {0};` will avoid problems later with uninitialized values.

Reply:
done, although the `getPw()` will not return success without initializing it.

BTW: this is a copy of other import utilities, there are lots of issues in utilities including leaks.

Reply:
Makes sense. Might be better to do a proper cleanup in the future.